src/index.js

```diff
@@ -359,6 +343,7 @@ class MiniCssExtractPlugin {
                     'var request = event && event.target && event.target.src || fullhref;',
                     'var err = new Error("Loading CSS chunk " + chunkId + " failed.\\n(" + request + ")");',
                     'err.request = request;',
+                    'delete installedCssChunks[chunkId]',
                     'reject(err);',
```

Comment:
Maybe also remove the failed style tags from the document.

Reply:
I have added code to remove the tag should onError be called, thanks for the suggestion